src/users/interfaces/user.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
```

Comment:
:wrench: 
Since this is a placeholder for the User Entity, let's also add a little remark to delete this file once the user entity is implemented:

```suggestion
//TODO: #24 - Remove this file and replace interface with User Entity Class
export interface User {
```